training/distributed_training/pytorch/model_parallel/gpt2/submit-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.            image_uri=f"855988369404.dkr.ecr.{region}.amazonaws.com/fewu-test:pytorch_1.7_tp_efa",
</pre><p>Update the image URI when it is available in a different location.</p><p> </p><p>Note that this will not work in China regions, which use <code>amazonaws.com.cn</code>  .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>